code/cutscene/movie.cpp

```diff
@@ -10,6 +10,7 @@
 
 
 #ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
```

Comment:
What's this for? I didn't see a reference to it in the changes.

Reply:
That prevents `windows.h` from including the winsock headers, which conflict with including them in psnet by way of `multi.h`.

Or as I like to say: "I hate Windows..." ;)